util/travis/clang-format-whitelist.txt

```diff
@@ -97,8 +97,10 @@ src/guiTable.h
 src/guiVolumeChange.cpp
 src/guiVolumeChange.h
 src/httpfetch.cpp
-src/hud.cpp
+src/client/hud.cpp
```

Comment:
please respect alphabetical order on this list. Why adding src/client/hud.h ? Linter is correct on it